app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self,
+                 name: str,
+                 appetit: int = 3,
+                 is_hungry: bool = True
+                 ) -> None:
+        super(Cat, self).__init__(name, appetit)
```

Comment:
no need to pass `Cat, self` to `super()` explicitly.
You forget about `is_hungry`

Reply:
Python automatically substitutes values, and who am I to argue with him? 
is_hungry added